02-express-tutorial/02-http-app.js

```diff
@@ -0,0 +1,52 @@
+const http = require("http");
+const { readFileSync } = require("fs");
+
+// get all files
+const homePage = readFileSync("./navbar-app/index.html");
+const homeStyles = readFileSync("./navbar-app/styles.css");
+const homeImage = readFileSync("./navbar-app/logo.svg");
+const homeLogic = readFileSync("./navbar-app/browser-app.js");
```

Comment:
I don't see the `navbar-app` directory in your repository -- did it get removed? That makes this file fail to execute (though all the code inside of it is correct).

Reply:
Yes please.

Thanks for catching this.

I will point them to the directory where the resources are.